packages/editor/src/client/api/index.ts

```diff
@@ -117,7 +117,7 @@ export type ArticleRevision = {
   canonicalUrl?: Maybe<Scalars['String']>;
   url: Scalars['String'];
   image?: Maybe<Image>;
-  authors: Array<Maybe<Author>>;
```

Comment:
We shouldn't have any changes in this file, shouldn't we?

Reply:
The 8.1.0 release was missing the generation of these files which is the reason why